modules/installing-ocp-agent-ibm-z.adoc

```diff
@@ -18,4 +18,44 @@ Depending on your {ibm-z-name} environment, you can choose from the following op
 [NOTE]
 ====
 Currently, ISO boot support on {ibm-z-name} (`s390x``) is available only for {op-system-base-full} KVM, which provides the flexibility to choose either PXE or ISO-based installation. For installations with z/VM and Logical Partition (LPAR), only PXE boot is supported.
-====
\ No newline at end of file
+====
+
+[id="networking-reqs-ibmz_{context}"]
+== Networking requirements for {ibm-z-title}
+
+In {ibm-z-title} environments, advanced networking technologies such as Original Storage Architecture (OSA), HiperSockets, and Remote Direct Memory Access (RDMA) over Converged Ethernet (RoCE) require specific configurations that deviate from the standard settings used in the Agent-Based Installer deployments. These overrides are necessary to accommodate their unique requirements and ensure a successful and efficient deployment on {ibm-z-title}.
+
+The following table lists the network devices that are supported for the network configuration override functionality:
+
+[cols="3,2,2,2,2", options="header"]
+|====
+| Network device 
+| z/VM  
+| KVM 
+| LPAR Classic 
+| LPAR Dynamic Partition Manager (DPM)
+
+| Original Storage Architecture (OSA) virtual switch 
+| Not needed
+| Not applicable
+| Not needed
+| Not needed
+
+| Direct attached OSA 
+| Supported
+| Supported only through a Linux bridge 
+| Supported
+| Not needed
+
+| RDMA over Converged Ethernet (RoCE) 
+| Not needed
+| Only through a Linux bridge 
```

Comment:
in line 46 the sentence is supported ony through a Linux bridge, is this the same here at line 52 and 58?
When reading this does it mean the ai.ip_cfg_override can be used in that context for KVM ?